pom.xml

```diff
@@ -90,6 +90,12 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
I discovered it's easier to just remove this junit:junit dependency - then you don't have to do the exclusion for hamcrest. Tested locally and it works. (Other dependencies have their own opinions on the junit and hamcrest versions, so it's easier to let them transitively pull in junit.)

Reply:
Thanks @chriskilding